_includes/samples/APIRequests/GenericBigWord.sol

```diff
@@ -3,6 +3,11 @@ pragma solidity ^0.8.7;
 
 import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
 
+/**
+ * Request testnet LINK and ETH here: https://faucets.chain.link/
```

Comment:
Getting testnet ETH with this link only works for kovan. Do you think it makes sense to clarify?

Reply:
I could change it to this:

`/**
 * Request testnet LINK (Arbitrum, Avalanche, Binance, Fantom, Kovan, Matic, POA, Rinkeby)
 * and ETH (Kovan) here: https://faucets.chain.link/
 * Find information on LINK Token Contracts here: https://docs.chain.link/docs/link-token-contracts/
 */`

lmk if this is better